server/package-lock.json

```diff
@@ -9,20 +9,11 @@
       "version": "1.0.0",
       "license": "ISC",
       "dependencies": {
-        "@types/bcrypt": "^5.0.2",
```

Comment:
all this deletions in this file will cause merge conflicts in the future, if you didn't change any package better to take this file from main 
`git checkout main server/package-lock.json` from your local branch + push should do the trick